nspl/ds/Set.php

```diff
@@ -8,6 +8,7 @@ class Set extends Collection
 {
     public function __construct(/* $e1, $e2, ..., $eN */)
     {
+        $this->array = [];
```

Comment:
I would like to use ```array()``` here instead of ```[]```. I still haven't used to square brackets for associative arrays and use them only for lists :)